frontend/tests/e2e_tests/run.js

```diff
@@ -313,7 +314,9 @@ const runCommand = (command, args = [], config, options = {}) =>
 
     child.on('error', error => {
       cleanup();
-      reject(error);
+      if (throwOnError) {
+        reject(error);
+      }
```

Comment:
@mzedel This is not (yet) the full fix, it still fails the cleanup with something like:
```
Cleanup💥 Failed to dump logs: Error: Command failed with exit code 1: (...)
cat: '/data/mender/deployment*.log': No such file or directory
```

I thought that the Promise won't throw errors, but just do the `resolve` or `reject` 🤔 

I can remove this whole "feature" about `throwOnError` if you want and find an alternate way to get the deployments logs.

Reply:
@mzedel  :dart: https://github.com/mendersoftware/mender-server/pull/1218/changes/b62320ffc46e5ff84d5a6a49001aef51d3a23848